common/views/components/ModalMoreFilters/ModalMoreFilters.tsx

```diff
@@ -16,7 +16,7 @@ import { AppContext } from '@weco/common/views/components/AppContext/AppContext'
 import CheckboxRadio from '@weco/common/views/components/CheckboxRadio/CheckboxRadio';
 import PlainList from '@weco/common/views/components/styled/PlainList';
 import { LinkProps } from '@weco/common/model/link-props';
-import DateRangeFilter from '@weco/common/views/components/SearchFilters/SearchFilters.DateRange';
+import DateRangeFilter from '@weco/common/views/components/SearchFilters/SearchFilters.DateRangeFilter';
```

Comment:
I wonder: should we export this (and all other components used externally) from `@weco/common/views/components/SearchFilters/index.tsx`, so external callers don't have to worry about the exact structure of this component?

Reply:
That's a nice idea, I'll look into that change